ompi/request/request.h

```diff
@@ -129,7 +134,7 @@ struct ompi_request_t {
     ompi_status_public_t req_status;            /**< Completion status */
     volatile void *req_complete;                /**< Flag indicating wether request has completed */
     volatile ompi_request_state_t req_state;    /**< enum indicate state of the request */
-    bool req_persistent;                        /**< flag indicating if the this is a persistent request */
+    bool req_flags;                             /**< flag indicating properties of the request (persistency, release resources) */
```

Comment:
`req_flags` is not a Boolean anymore. I suggest to use int here instead (won't change the layout o the struct).